packaging/debian/copyright

```diff
@@ -0,0 +1,35 @@
+Upstream Homepage:
+    https://www.gen2vdr.de/wirbel/w_scan/index2.html
+
+Upstream Author:
+    Winfried Koehler <w_scan AT gmx-topmail DOT de>
```

Comment:
You should add my info here, since the original author cannot be expected to maintain this fork.